api/test/fixtures/users.json

```diff
@@ -0,0 +1,20 @@
+[{
+    "User": [
+        {
+            "email"   : "test_user@email.com",
+            "name"    : "Test User",
+            "username": "testuser",
+            "password": "testuser"
+        }
+    ]
+},
+{
```

Comment:
Any reason for separating `testuser` and `anotheruser` import into two batches?

Reply:
Just blind.